otaclient/ota_proxy/orm.py

```diff
@@ -15,19 +15,30 @@
 
 from __future__ import annotations
 
-from abc import ABC
 from dataclasses import asdict, astuple, dataclass, fields
 from io import StringIO
-from typing import (TYPE_CHECKING, Any, Callable, Dict, Generic, List,
-                    Optional, Tuple, Type, TypeVar, Union, overload)
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    Dict,
+    Generic,
+    List,
+    Optional,
+    Tuple,
+    Type,
+    TypeVar,
+    Union,
+    overload,
+)
 
 from typing_extensions import Self, dataclass_transform
 
 if TYPE_CHECKING:
     import sqlite3
 
 
-class NULL_TYPE(ABC):
+class NULL_TYPE:
     """Singleton for NULL type."""
 
```

Comment:
I used to bind the real `None` type as subclass to `NULL_TYPE`, but later remove this logic.
The ABC base class here is the leftover of the above removing.